packages/react-server/src/ReactFlightStreamer.js

```diff
@@ -0,0 +1,144 @@
+/**
```

Comment:
just curious, why put this in react-server instead of react-flight?

Reply:
Mostly because both Fizz and Flight depend on ReactServerHostConfig/ReactServerFormatConfig so it lets me avoid hoisting those concerns to a third package. It's quite likely that the two servers would be running in the same machine and interleaved. So you likely want to version them together too.

It gets awkward now that I'm adding a client API too which needs its own package or join `react-reconciler`.

It's not that important though since these are all not the encouraged APIs other than for custom builds. The public API is the `react-dom` entry points. We don't even own the `react-server` npm name so will likely change.